src/common/latest/run.ts

```diff
@@ -1,13 +1,16 @@
 import * as tl from "azure-pipelines-task-lib/task";
+import { log, LogLevel, setEndpointType } from "./helpers/logging";
 import { EndpointType } from "./sonarqube/Endpoint";
 
-export type TaskJob = (endpoint: EndpointType) => Promise<void>;
+export type TaskJob = (endpointType: EndpointType) => Promise<void>;
+
+export async function runTask(fun: TaskJob, taskName: string, endpointType: EndpointType) {
+  setEndpointType(endpointType);
 
-export async function runTask(fun: TaskJob, taskName: string, endpoint: EndpointType) {
   try {
-    await fun(endpoint);
+    await fun(endpointType);
```

Comment:
how does the user set the log level override for DEBUG (verbose)

I would expect something like:
https://github.com/SonarSource/sonar-scanner-npm/blob/master/src/scan.ts#L50C28-L50C57

I see that by default the user sees INFO/WARN/ERROR

Reply:
They pass it through Azure DevOps when they check this
![image](https://github.com/user-attachments/assets/63f8fbbe-0616-41aa-b290-4e3092e61c2b)

I didn't find a way to set the pipeline in debug mode from within a task  🤔so the user can choose 2 log levels:
- debug, he'll see everything
- default, he'll see info, warn and errors

That's why I also dropped the TRACE logs (compared to the NPM implem) WDYT?

Reply:
ah great! I apparently always missed this flag 😅